presto-docs/src/main/sphinx/admin/properties-session.rst

```diff
@@ -601,3 +601,21 @@ The corresponding configuration property is :ref:`admin/properties:\`\`materiali
 
 Enable optimization to combine multiple :func:`!approx_distinct` function calls on expressions
 of the same type into a single aggregation using ``set_agg`` with array operations (``array_constructor``, ``array_transpose``).
+
+``optimizer.merge_max_by_aggregations``
```

Comment:
Should also work the same way for min_by aggs. Change the option name to: ...max_by_and_min_by_...